### ISSUE TITLE
[TW-1629] Descriptions often get overwritten with "( or )"

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1653]
_Jeremy John Reeder on 2015-07-07T19:35:31Z says:_ 

 When I try to modify a field of every task matching a query, sometimes it works right but other times the descriptions of all matching tasks get overwritten with the text `( or )`. To work around this, I have to embed a call to an ID-listing command such as `ids`.

For example:
{code}
$ task proxy:Dillon modify proxy:Dylan
Modifying task 1243 '( or )'.
Modifying task 1244 '( or )'.
Modifying task 1245 '( or )'.
Modified 3 tasks.
{code}

Workaround:
{code}
$ task $(task proxy:Dillon ids) modify proxy:Dylan
Modifying task 1243 'Get WiFi logs'.
Modifying task 1244 'Get VPN logs'.
Modifying task 1245 'Get email logs'.
{code}

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2015-07-07T19:35:31Z
Modified: 2015-08-03T20:46:34Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2015-07-08T01:23:32Z says:_ 

 I'd be interested in seeing debug output from that failing run, which means "rc.debug.parser=3", but of course that requires you to sacrifice time and a data restore.  But clearly it is building the filter incorrectly, but we don't know how yet.

**Comment by taskwarrior** [comment]
_Jeremy John Reeder on 2015-07-21T19:56:20Z says:_ 

 I attached a log of the result of the following command, which exhibited about the same behavior. That is, the description became "( ) ( )".
{code}
task rc.debug.parser=3 rc.context=dom 1063 mod loc:Somewhere
{code}
I've found that it does _not_ happen when the context is unset:
{code}
task rc.debug.parser=3 rc.context= 1063 mod loc:Somewhere
Modifying task 1063 'Sell juicer'.
Modified 1 task.
Project 'dom' is 88% complete (22 of 187 tasks remaining).
{code}
